bin/llvm-kompile-testing

```diff
@@ -11,7 +11,17 @@ trap 'rm -rf "$dt_dir"' INT TERM EXIT
 definition=$(realpath "$1")
 mode="$2"
 shift; shift
-( cd "@PROJECT_SOURCE_DIR@"/matching && mvn exec:java -Dexec.args="$definition qbaL $dt_dir 1" -q )
+
+matching_dir="@PROJECT_SOURCE_DIR@/matching"
+installed_jar="$matching_dir/target/llvm-backend-matching-1.0-SNAPSHOT-jar-with-dependencies.jar"
+
+if [ ! -f "$installed_jar" ]; then
+  echo "Pattern matching compiler isn't installed!"
+  echo "  looking for a jar file at $installed_jar"
+  exit 1
```

Comment:
Why are we exiting here?

Reply:
Because the pattern matching compiler jar isn't installed; how would you propose we continue execution?

Reply:
The message above makes me feel that the script will look for the jar and not finish its execution...
I think I would create an else branch to build it as we were before, no?

Reply:
Alright! Makes sense!